tests/functional/object/mcg/test_noobaa_bucket_error_state_mode.py

```
@@ -0,0 +1,560 @@
+            "namespace_policy_dict": {
+                "type": "Single",
+                "namespacestores": ns_stores,
+            },
+        }
+        ns_bucket = bucket_factory(
+            amount=1,
+            interface=bucketclass_dict["interface"],
+            bucketclass=bucketclass_dict,
+        )[0]
+
+        def cleanup():
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `cleanup`

Add return type annotation: `None`

(ANN202)